vncviewer/DesktopWindow.h

```diff
@@ -26,7 +26,6 @@
 
 #include <rfb/Rect.h>
 #include <rfb/Pixel.h>
-
```

Comment:
Please keep this spacing. It provides some group of internal and external stuff.